frontend/src/components/Agreements/AgreementsTable/AgreementTableRow.jsx

```diff
@@ -167,16 +212,36 @@ export const AgreementTableRow = ({ agreement }) => {
                         {agreementCreatedOn}
                     </dt>
                 </dl>
+                <dl
+                    className="font-12px width-mobile"
+                    style={{ marginLeft: "40px" }}
+                >
+                    <dt className="margin-0 text-base-dark">Description</dt>
+                    <dd
+                        className="margin-0"
+                        style={{ maxWidth: "400px" }}
+                    >
+                        {agreementDescription ? agreementDescription : "No description created."}
+                    </dd>
+                </dl>
                 <dl
                     className="font-12px"
-                    style={{ marginLeft: "9.0625rem" }}
+                    style={{ marginLeft: "50px" }}
```

Comment:
50 px = 3.125 rem if you want to keep the units

Reply:
done